src/SoundList.js

```diff
@@ -9,7 +9,35 @@ function timeConvert(minutes) {
   return `${(min < 10 ? "0" : "") + min}:${secs < 10 ? "0" : ""}${secs}`;
 }
 
-const SoundList = ({ tracks, header, onSoundClick, className = "" }) => {
+// Psuedo code:
+// function highlightSound(track, sound) {
+//   let packSoundArr = pack sounds array;
+//   let similarSoundArr = similar array;
+//   for (i = 0; i<=packSoundArr.length; i++) {
+//     for (j=0; j <= similarSoundArr; j++) {
+//       if (track.name === sound.name) {
+//         keep background color set to secondary color regardless of hover class
+//       }
+//     }
+//   }
+// }
+
+const SoundList = ({
+  tracks,
+  header,
+  onSoundClick,
+  currentTrackId,
+  className = "",
+}) => {
+  const selectedTrackClassName =
+    "group bg-secondary text-primary transition duration-150 ease-in-out border border-solid border-secondary p-2 space-x-2 flex items-center";
+  const unSelectedTrackClassName =
+    "group hover:bg-secondary hover:text-primary transition duration-150 ease-in-out border border-solid border-secondary p-2 space-x-2 flex items-center";
+  const selectedSoundDataClassName =
```

Comment:
@akshara-sun I'm going to answer your question here since I think it's easier to have the discussion adjacent to the code in question.

> I have a couple of questions @hamirmahal :
> const selectedTrackClassName - does this refer to the track inside of the SoundList component?
> const selectedSoundDataClassName - does this refer to the selected sound on the Sound Page?

In general, you can tell what the lefthand side of an assignment refers to by checking the righthand side of that assignment.

For instance, for `const selectedTrackClassName = "group ... items-center";`, the righthand side of the assignment is a String. From that, we can tell that `selectedTrackClassName` must refer to something that is a String.

Similarly, for `const selectedSoundDataClassName = "w-6/12 ... sm:hover:opacity-0";`, the righthand side of that assignment is a String, so `selectedSoundDataClassName` should also refer to a String.

`selectedTrackClassName` is a String that holds information relevant to the styling for a specific track in `tracks`, which the `<SoundList />` component receives as a prop from a parent in the component tree. And `selectedSoundDataClassName` is a String that holds information relevant to the styling for the icons for a sound's data, like the number of downloads, length, and number of ratings.

Reply:
got it, thank you

Reply:
You're welcome!